test/repository-tests.jl

```diff
@@ -57,7 +57,7 @@ domain = getRepositoryDomain(token, appKey=appKey)
 @test haskey(domain["custom_timers"], "ResourceTimer")
 
 @test domain["custom_timers"]["ResourceTimer"]["index"] == 0
-@test domain["custom_timers"]["ResourceTimer"]["fieldname"] == "timers_custom0"
+@test domain["custom_timers"]["ResourceTimer"]["fieldname"] == "customtimer0"
 @test domain["custom_timers"]["ResourceTimer"]["mpulseapiname"] == "CustomTimer0"
 
```

Comment:
need to add tests for alerts

Reply:
mPulse doesn't seem to give the ability to add alerts on the `SOASTA mPulseAPI test` tenant

Reply:
added tests for test alert